scripts/images/rancher-images.txt

```diff
@@ -21,3 +21,7 @@ docker.io/rancher/shell:v0.1.16
 docker.io/rancher/shell:v0.1.8
 docker.io/rancher/system-agent:v0.2.3-suc
 docker.io/rancher/system-upgrade-controller:v0.8.1
+docker.io/rancher/mirrored-banzaicloud-fluentd:v1.14.6-alpine-5
+docker.io/rancher/mirrored-banzaicloud-logging-operator:3.17.7
+docker.io/rancher/mirrored-fluent-fluent-bit:1.9.3
+docker.io/rancher/mirrored-jimmidyson-configmap-reload:v0.4.0
```

Comment:
Missing `banzaicloud/eventrouter:v0.1.0` image in the air-gap environment.
<img width="785" alt="Screen Shot 2022-09-17 at 12 20 02 AM" src="https://user-images.githubusercontent.com/1691518/190685393-57d094d9-bc40-469b-8f6b-49f4ec83b52f.png">

Reply:
Good catch.  added and tested.

```
harvester-installer make:
....
docker image pull --quiet docker.io/banzaicloud/eventrouter:v0.1.0
docker.io/banzaicloud/eventrouter:v0.1.0
...
docker image pull --quiet docker.io/kubevip/kube-vip-cloud-provider:v0.0.1
docker.io/kubevip/kube-vip-cloud-provider:v0.0.1
...
```

Reply:
The POD's resource limits are also added.

```
kk get pods -n cattle-logging-system harvester-default-event-tailer-0 -o YAML
..
spec:
  containers:
  - image: banzaicloud/eventrouter:v0.1.0
    imagePullPolicy: IfNotPresent
    name: event-tailer
    resources:
      limits:
        cpu: 100m
        memory: 100Mi
      requests:
        cpu: 20m
        memory: 20Mi

```